torch_geometric/nn/models/mlp.py

```diff
@@ -77,14 +81,14 @@ def __init__(
         hidden_channels: Optional[int] = None,
         out_channels: Optional[int] = None,
         num_layers: Optional[int] = None,
-        dropout: float = 0.,
+        dropout: Optional[Union[List[float], float]] = 0.,
         act: Union[str, Callable, None] = "relu",
         act_first: bool = False,
         act_kwargs: Optional[Dict[str, Any]] = None,
         norm: Union[str, Callable, None] = "batch_norm",
         norm_kwargs: Optional[Dict[str, Any]] = None,
         plain_last: bool = True,
-        bias: bool = True,
+        bias: Optional[Union[List[bool], bool]] = True,
```

Comment:
```suggestion
        bias: Union[List[bool], bool] = True,
```
The implementation does not handle these being None so I'd suggest just not marking as optional